account_financial_risk/models/account_invoice.py

```diff
@@ -59,27 +60,39 @@ def risk_exception_msg(self):
             exception_msg = _("This invoice exceeds the financial risk.\n")
         return exception_msg
 
-    def action_post(self):
+    def _post(self, soft=True):
         if (
             self.env.context.get("bypass_risk", False)
             or self.company_id.allow_overrisk_invoice_validation
         ):
-            return super().action_post()
+            return super()._post(soft)
```

Comment:
Please always past keyword arguments as such:

```suggestion
            return super()._post(soft=soft)
```

Reply:
but `_post` only has one parameter, it seems doing this is irrelevant :S